pvlib/singlediode.py

```diff
@@ -351,9 +431,17 @@ def bishop88_mpp(photocurrent, saturation_current, resistance_series,
         (a-Si) modules that is the product of the PV module number of series
         cells ``Ns`` and the builtin voltage ``Vbi`` of the intrinsic layer.
         [V].
-    method : str
-        one of two optional search methods: either ``'brentq'``, a reliable and
-        bounded method or ``'newton'`` which is the default.
+    breakdown_factor : numeric, default 0
+        fraction of ohmic current involved in avalanche breakdown :math:`a`.
+        Default of 0 excludes the reverse bias term from the model. [unitless]
+    breakdown_voltage : numeric, default -5.5
+        reverse breakdown voltage of the photovoltaic junction :math:`V_{br}`
+        [V]
+    breakdown_exp : numeric, default 3.28
+        avalanche breakdown exponent :math:`m` [unitless]
+    method : str, default 'newton'
+        must be either ``'newton'`` or ``'brentq'``. If ``'breakdown_factor'``
```

Comment:
Does `breakdown_factor` need `'` around it?

Reply:
Replacing with 
``` 
   method : str, default 'newton'
        must be either ``'newton'`` or ``'brentq'``. If ``breakdown_factor``
        is not 0, ''method'' must be ``'newton'``.
```